devices/esp32-c3/main/build/CMakeFiles/CMakeConfigureLog.yaml

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
same here, lets add to gitignore

Reply:
I deleted the whole build directory and added that to .gitignore